src/ui/header/navigation.tsx

```
@@ -0,0 +1,173 @@
+
+export default async function Navigation() {
+  const { headerMenu } = await getSite();
+
+  return (
+    <NavigationMenu>
+      <NavigationMenuList>
+        {(headerMenu as HeaderMenu)?.items?.map((item, key) => {
+          switch (item._type) {
+            case 'link':
+              return (
+                <NavigationMenuItem key={key}>
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)